### PR TITLE
Fix installation instruction for extractorapp and atlas database

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

**Comment by Vampouille** [comment]
we also need postgis extension !

**Comment by Vampouille** [comment]
> we also need postgis extension !

Fixed by  	fec2e25

File: docs/setup/postgresql.md [hunk 1]
```diff
@@ -52,10 +52,25 @@ If the **security proxy** is deployed and ```shared.ogc.statistics.activated```
 psql -d georchestra -f postgresql/05-ogc-server-statistics.sql
 ```
 
+### Extractorapp schema
+
+If the **extractor app** is deployed:
+```
+psql -d georchestra -f postgresql/01-create-extension.sql
```
Comment (fvanderbiest) [inline docs/setup/postgresql.md]: Really not sure we need `01-create-extension.sql` here.

Comment (Vampouille) [inline docs/setup/postgresql.md]: Table `extractorapp.extractor_layer_log` contains geometry :

https://github.com/georchestra/georchestra/blob/16.12/postgresql/09-extractor-app.sql#L20

Comment (fvanderbiest) [inline docs/setup/postgresql.md]: OK, I did not see that. Thanks !

**Pull request merged**